src/wp-includes/media.php

```diff
@@ -275,26 +275,29 @@ function image_downsize( $id, $size = 'medium' ) {
  * Register a new image size.
  *
  * @since 2.9.0
+ * @since 6.1.0 Add the $output_mimes parameter.
  *
  * @global array $_wp_additional_image_sizes Associative array of additional image sizes.
  *
- * @param string     $name   Image size identifier.
- * @param int        $width  Optional. Image width in pixels. Default 0.
- * @param int        $height Optional. Image height in pixels. Default 0.
- * @param bool|array $crop   Optional. Image cropping behavior. If false, the image will be scaled (default),
- *                           If true, image will be cropped to the specified dimensions using center positions.
- *                           If an array, the image will be cropped using the array to specify the crop location.
- *                           Array values must be in the format: array( x_crop_position, y_crop_position ) where:
- *                               - x_crop_position accepts: 'left', 'center', or 'right'.
- *                               - y_crop_position accepts: 'top', 'center', or 'bottom'.
+ * @param string     $name         Image size identifier.
+ * @param int        $width        Optional. Image width in pixels. Default 0.
+ * @param int        $height       Optional. Image height in pixels. Default 0.
+ * @param bool|array $crop         Optional. Image cropping behavior. If false, the image will be scaled (default),
+ *                                 If true, image will be cropped to the specified dimensions using center positions.
+ *                                 If an array, the image will be cropped using the array to specify the crop location.
+ *                                 Array values must be in the format: array( x_crop_position, y_crop_position ) where:
+ *                                     - x_crop_position accepts: 'left', 'center', or 'right'.
+ *                                     - y_crop_position accepts: 'top', 'center', or 'bottom'.
+ * @param bool       $output_mimes Whether to output secondary mimes for this image size. Default is false.
  */
-function add_image_size( $name, $width = 0, $height = 0, $crop = false ) {
+function add_image_size( $name, $width = 0, $height = 0, $crop = false, $output_mimes = false ) {
```

Comment:
`$output_mimes` is a bit generic. This is only about additional / secondary MIME types. Maybe we should call it `$additional_mimes`?

Reply:
updated